src/createContext.ts

```diff
@@ -51,44 +53,50 @@ export const defineSymbol = (context: Context, name: string, value: Value) => {
   })
 }
 
+export const defineBuiltin = (context: Context, name: string, f: Function) => {
```

Comment:
Should we just have one function called `defineBuiltin`? then if it is a function we add the toString, if not we leave it as it is. This would simplify the logic of externalSymbols and importing math libraries.